tox.ini

```diff
@@ -0,0 +1,49 @@
+[tox]
+envlist = clean,py3,flake8
+
+
+[pytest]
+testpaths = tests
+
+
+[coverage:report]
+exclude_lines =
+    pragma: no cover
+
+    def __repr__
+    if self.\.debug
+
+    raise AssertionError
+    raise NotImplementedError
+    if __name__ == .__main_.:
+
+
+[testenv]
+deps = pytest>=3.5.0
+       flake8
+       jinja2
+       pytest-cov
+       selenium
+       coverage
+       pydantic
+
+commands = coverage run --source=pytest_webdriver_recorder -m pytest
+           coverage report --fail-under=99
+
+
+[testenv:clean]
+skip_install = True
+deps = coverage
+commands = coverage erase
+
+
+[testenv:flake8]
+skip_install = true
+deps = flake8
+commands = flake8 {toxinidir}/pytest_webdriver_recorder
+
+
+[flake8]
+max-line-length = 119
```

Comment:
Need to confirm whether others on the team are all right with breaking the `79` convention.